packages/main/src/auth/useChannel.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+const useChannel = channel_id => {
+  const BASE_API_URL = "https://chat.zuri.chat";
+  const token = sessionStorage.getItem("token");
+  //const member_id = localStorage.getItem("user");
+  const org_id = localStorage.getItem("currentWorkspace");
+  const member = JSON.parse(sessionStorage.getItem("user")) || null;
+  const member_id = member.id;
+  //const room_id =member.id
+
+  const createChannel = data => {
+    //const aisosa= `${BASE_API_URL}/api/v1/org/${org_id}/members/${member_id}`
+    //console.log(data,aisosa, "here");
+    axios.post(
+      `${BASE_API_URL}/api/v1/org/${org_id}/members/${member_id}/rooms`,
+      data,
+      {
+        headers: {
+          //Authorization: `Bearer ${token}`,
```

Comment:
Remove commented out code, also are you sure token doesn't need to be sent in the request?

Reply:
deleted file